map_claims_test.go

```diff
@@ -134,3 +134,56 @@ func TestMapClaimsVerifyExpiresAtExpire(t *testing.T) {
 		t.Fatalf("Failed to verify claims, wanted: %v got %v", want, (got == nil))
 	}
 }
+
+func TestMapClaims_ParseString(t *testing.T) {
+	type args struct {
+		key string
+	}
+	tests := []struct {
+		name    string
+		m       MapClaims
+		args    args
+		want    string
+		wantErr bool
+	}{
+		{
+			name: "missing key",
```

Comment:
nit, i wish we established a pattern where we named these "missing_key" or "missing-key" so the names are copy/pasteable.

that way it's easy to do -run=TestMapClaims_ParseString/{name} without having to deal with regex or manually concatenate the name.

(this isn't the best example, but still).

Otherwise thanks for adding more tests!